packages/graphql-language-service-server/src/__tests__/findGraphQLTags-test.ts

```diff
@@ -165,6 +165,32 @@ query Test {
     expect(contents[0].template).toEqual(` query {} `);
   });
 
+  it('finds queries in tagged templates in Vue SFC using <script setup>', async () => {
+    const text = `
+<script setup lang="ts">
+gql\`
+query {id}
+\`;
+</script>
+`;
+    const contents = findGraphQLTags(text, '.vue');
+    expect(contents[0].template).toEqual(`
+query {id}`);
+  });
+
+  it('finds queries in tagged templates in Vue SFC using normal <script>', async () => {
+    const text = `
+<script lang="ts">
+gql\`
+query {id}
+\`;
+</script>
+`;
+    const contents = findGraphQLTags(text, '.vue');
+    expect(contents[0].template).toEqual(`
+query {id}`);
+  });
+
```

Comment:
thank you for adding these tests!!

reading more about vue, i wonder if there are cases where .vue files are plain typescript/javascript and whether we should test for those - i `think` i see where you account for this in `findGraphQLTags`? If i understand correctly, render functions may be a case of this? or do these files have standard js/ts files in vue? https://vuejs.org/guide/extras/render-function.html#creating-vnodes

Reply:
As far as I know there is no such cases. Though we can write Vue components as non-SFC files using render functions, they should have a .js, .jsx, .ts or .tsx extension.

However thanks to your comment I realized I forgot to handle JSX in SFC.  `@vue/compiler-sfc` seems to supports JSX out of the box, so I added a test for it.

Reply:
i think that's what i meant, rad thanks!